POEApi.Model/ProxyMapper.cs

```diff
@@ -298,5 +299,18 @@ internal static List<Tab> GetTabs(List<JSONProxy.Tab> tabs)
                 throw;
             }
         }
+
+        public static ChargeInfo GetCharges(List<JSONProxy.Property> list)
+        {
+            var chargeSize = list.Find(p => p.Name == CHARGES);
+            if (chargeSize == null)
+                return new ChargeInfo(0, 0);
+
+            var qty = (object[]) chargeSize.Values[0];
```

Comment:
I have checked out the JSon that comes back.

It's a little bit of an odd way of doing it and it's not exactly clear if the first set or the last set are the number of charges or the max charges.

```
	{
		"verified": false,
		"w": 1,
		"h": 1,
		"ilvl": 69,
		"icon": "https:\/\/web.poecdn.com\/image\/Art\/2DItems\/Currency\/Classic\/Prophecy.png?scale=1&w=1&h=1&v=ec3aac908a0be262b312c01c88545e7f3",
		"league": "SSF Legacy",
		"id": "8ab56a1f74390a4454b93fb7e24d42d6243828c7e157d5fc48840040316e9c64",
		"sockets": [],
		"name": "",
		"typeLine": "<<set:MS>><<set:M>><<set:S>>Ample Prophecy Leaguestone of Silver",
		"identified": true,
		"corrupted": true,
		"lockedToCharacter": false,
		"properties": [{
			"name": "Currently has %0 of %1 Charges",
			"values": [["5",
			0],
			["5",
			0]],
			"displayMode": 3
		}],
		"implicitMods": ["Next Area will contain a Silver Coin"],
		"explicitMods": ["Next Area will contain an additional Silver Coin",
		"Monsters with Silver Coins in next Area will drop an additional Silver Coin"],
		"descrText": "Right-click to open your Legacy Panel and place a Leaguestone in an empty slot. It will apply to the next new area you enter.",
		"frameType": 1,
		"x": 1,
		"y": 0,
		"inventoryId": "Stash8",
		"socketedItems": []
	}
```

As you can see the properties are a pairing of string and int. So it's an array with mixed types. I can see the concern with this loose object[] cast but I can't think of a better way. I'm open to suggestions. Atleast this method is not wrapped with a try catch and will return a "Sensible" object.